src/pfb/exporters/tsv.py

```diff
@@ -6,6 +6,14 @@
 
 from ..cli import to_command
 
+plural_parents = {
+    "subjects":"subject",
```

Comment:
does this repo not run `wool` checks? It must not, can you run the `black` style formatter over this?

Reply:
And if you have time, I wouldn't mind if you added the wool check, but I understand that's somewhat out of scope